### PR TITLE
DAPP-394 & DAPP-393: Transactions Table Tests

### PR DESCRIPTION
**Pull request by lukorito**
## Description

Add tests for transactions table for desktop, table and mobile
Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?
 make file_test

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings
- [ ] I have added screenshots

### CODE REVIEW

File: lib/transactions/widgets/paginated_transaction_table.dart [hunk 1]
```diff
@@ -28,6 +29,7 @@ class PaginatedTransactionTable extends HookConsumerWidget {
     final isBiggerScreen = MediaQuery.of(context).size.width == 1920;
     final isTablet = ResponsiveBreakpoints.of(context).equals(TABLET);
     final isMobile = ResponsiveBreakpoints.of(context).equals(MOBILE);
+    print("called");
```
Comment (KalervoHyyppa) [inline lib/transactions/widgets/paginated_transaction_table.dart:32]: Clean Up

**Review by KalervoHyyppa — CHANGES_REQUESTED** [review]
One super small request

File: lib/transactions/widgets/paginated_transaction_table.dart (hunk 1 above)
Comment (lukorito) [reply lib/transactions/widgets/paginated_transaction_table.dart]: Sorted @KalervoHyyppa

**Review by KalervoHyyppa — APPROVED** [review]
lgtm

**Pull request merged**